packages/lib-react-components/src/Media/components/Data/components/DataSeriesPlot/DataSeriesPlot.js

```diff
@@ -0,0 +1,288 @@
+          strokeWidth={2}
+          x1={xErrorBarPoints.x1}
+          x2={xErrorBarPoints.x2}
+          y1={cy}
+          y2={cy}
+        />}
+      {y_error &&
+        <line
+          stroke={lighten(0.25, glyphColor)}
+          strokeWidth={2}
+          x1={cx}
+          x2={cx}
+          y1={yErrorBarPoints.y1}
+          y2={yErrorBarPoints.y2}
+        />}
+      <GlyphComponent
+        data-x={x}
+        data-y={y}
+        left={cx}
+        size={dataPointSize}
+        top={cy}
+        fill={glyphColor}
+        stroke={'black'}
+      />
+    </g>
+  )
+}
+
+function DataSeriesPlot({
+  alt = '',
+  backgroundColor = '',
+  chartOptions = CHART_OPTIONS,
+  children,
+  data,
+  dataPointSize = 25,
+  parentHeight,
+  parentWidth,
+  transformMatrix = TRANSFORM_MATRIX,
+  transform,
+  underlays = []
+}) {
+  const {
+    dark,
+    global: {
+      colors = {}
+    }
+  } = useTheme()
+
+  const {
+    invertAxes = INVERT_AXES,
+    margin = MARGIN,
+    padding = PADDING,
+    xAxisLabel,
+    xAxisLabelOffset,
+    yAxisLabel,
+    yAxisLabelOffset
+  } = chartOptions
+
+  const rangeParameters = {
+    invertAxes,
+    margin,
+    padding,
+    parentHeight,
+    parentWidth,
+    tickDirection: 'outer'
+  }
+
+  const leftPosition = margin.left
+  const topPosition = margin.top
+
+  let background 
+  if (backgroundColor) {
+    background = backgroundColor
+  } else {
+    background = getZoomBackgroundColor(dark, false, colors)
+  }
+
+  const sortedDataPoints = getDataPoints(data)
+  const xScaleTransformed = transformXScale(data, transformMatrix, rangeParameters)
+
+  const yScaleTransformed = transformYScale(data, transformMatrix, rangeParameters)
+
+  const clipPathId = cuid()
+  const plotHeight = parentHeight - margin.bottom - margin.top
+  const plotWidth = parentWidth - margin.right - margin.left
+
+  let underlayParameters = []
+  if (underlays.length > 0) {
+    underlayParameters = underlays.map((underlay) => {
+      const { fill, startPosition, xAxisWidth } = underlay
+      const left = xScaleTransformed(startPosition)
+      const width = xScaleTransformed(0) - xScaleTransformed(-xAxisWidth)
+      return { fill, left, width }
+    })
+  }
+  return (
+    <svg
+      role='img'
+      aria-label={alt}
+      height='100%'
+      width='100%'
+    >
+      <rect fill={background} />
+      <clipPath id={`scatter-plot-${clipPathId}`}>
+        <rect
+          height={plotHeight}
+          width={plotWidth}
+        />
+      </clipPath>
+      <Group
+        clipPath={`url(#scatter-plot-${clipPathId})`}
+        left={leftPosition}
+        top={topPosition}
+      >
+        <rect
+          borderColor={(dark) ? colors['light-5'] : colors['dark-5']}
+          fill={(dark) ? colors['light-3'] : colors['neutral-6']}
+          height={plotHeight}
+          left={leftPosition}
+          top={topPosition}
+          underlayParameters={underlayParameters}
+          width={plotWidth}
+        />
```

Comment:
The `borderColor` and `underlayParameters` props throw console warnings because < rect > is  a DOM element. I recommend renaming to not clutter the console in a dev environment.

<img width="396" alt="Screenshot 2023-03-24 at 7 48 02 AM" src="https://user-images.githubusercontent.com/23665803/227525709-212d269a-c82e-4130-b9ec-82e6b6782d45.png">
<img width="403" alt="Screenshot 2023-03-24 at 7 47 46 AM" src="https://user-images.githubusercontent.com/23665803/227525714-35763a84-ecdb-410d-beae-74a36bd06bb4.png">